client/sites-dashboard-v2/dotcom-style.scss

```diff
@@ -316,6 +319,23 @@
 			}
 		}
 	}
+	.item-preview__content {
+
+		> * {
+			padding: 0;
+			overflow-y: initial;
+			max-height: initial;
+			max-width: 1400px;
+		}
+
+		@media (max-width: $break-large) {
+			padding: 24px 24px 8px;
+		}
+
+		.main {
```

Comment:
@fushar I added this to fix the alignment on larger screen sizes. See https://github.com/Automattic/wp-calypso/pull/90320#issuecomment-2096447300 Feel free to change it if there's a better way!

Reply:
Thanks! Changed it to https://github.com/Automattic/wp-calypso/pull/90320/commits/15713fbad0fa81a0e01570b861fdea3791e477db so that it affects all children of the tab content :)